utils/data_retrieval.py

```diff
@@ -25,6 +26,59 @@ def trim_base_from_uri(uri_to_trim):
     return uri_to_trim.replace(TILED_BASE_URI, "")
 
 
+def get_processed_experiment_names():
+    try:
+        client = from_uri(TILED_URI, api_key=TILED_API_KEY)
+        processed_list_of_experiments = client["processed"].keys()
+        list_of_experiments = [
+            experiment for experiment in processed_list_of_experiments
+        ]
```

Comment:
I just learned we can just to `list(client)`, see [Tiled Navigation Tutorial](https://blueskyproject.io/tiled/tutorials/navigation.html) to get all keys of a client, so we could shorten this to a sections.

In the current version, processed contains more than experiments, e.g. there will also be a `mask` sections and each folder will have one too, so we may have to think about created a dedicated place for experiment summaries.